model/account.rb

```diff
@@ -37,6 +36,34 @@ def create_project_with_default_policy(name, default_policy: true)
     project
   end
 
+  def hyper_tag(project)
+    AccessTag.where(project_id: project.id, hyper_tag_id: id).first
+  end
+
+  def associate_with_project(project)
+    return if project.nil?
+
+    # Temporary code to ensure that project_id column already set,
+    # except for Account which doesn't have the column
+    # :nocov:
+    case self
+    when Account
+      nil
+    when Project
```

Comment:
Since we moved it to the `Account` model, self can't be `Project`

Reply:
Good point, I'll fix that.